charts/gateway-operator/Chart.yaml

```diff
@@ -8,10 +8,10 @@ maintainers:
 name: gateway-operator
 sources:
   - https://github.com/Kong/charts/tree/main/charts/gateway-operator
-version: 0.4.9
+version: 0.4.10-rc.1
 appVersion: "1.4"
 annotations:
-  artifacthub.io/prerelease: "false"
+  artifacthub.io/prerelease: "true"
```

Comment:
We might want to leave a comment here to change this when a non RC is going to be released.

Reply:
Yep, it's not needed - reverting